tests/integration/targets/vm_snapshot_info/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+
+
+  # TODO: update tests to work with other params - vm_name, serial
+  block:
+    # --------------------------------------------------------
+
+    - name: List all VM snapshots - API
+      scale_computing.hypercore.api:
+        action: get
+        endpoint: /rest/v1/VirDomainSnapshot
+      register: vm_snapshots_api
+    - ansible.builtin.set_fact:
+        existing_snapshot_label: "{{ vm_snapshots_api.record[0].label }}"
+    - ansible.builtin.debug:
+        var: existing_snapshot_label
+
+    # --------------------------------------------------------
+
+    - name: List all VM snapshots with snapshot label "{{ existing_snapshot_label }}"
+      scale_computing.hypercore.vm_snapshot_info:
+        label: "{{ existing_snapshot_label }}"
+      register: vm_snapshots
+    - ansible.builtin.debug:
+        var: vm_snapshots.records
+    - ansible.builtin.assert:
+        that:
+          - vm_snapshots.records != []
```

Comment:
Test also change == False and other `obvious and we never had a typo there` fields